regression-test/suites/materialized_view_p0/testAggQueryOnAggMV2/testAggQueryOnAggMV2.groovy

```diff
@@ -0,0 +1,65 @@
+// Licensed to the Apache Software Foundation (ASF) under one
+// or more contributor license agreements.  See the NOTICE file
+// distributed with this work for additional information
+// regarding copyright ownership.  The ASF licenses this file
+// to you under the Apache License, Version 2.0 (the
+// "License"); you may not use this file except in compliance
+// with the License.  You may obtain a copy of the License at
+//
+//   http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing,
+// software distributed under the License is distributed on an
+// "AS IS" BASIS, WITHOUT WARRANTIES OR CONDITIONS OF ANY
+// KIND, either express or implied.  See the License for the
+// specific language governing permissions and limitations
+// under the License.
+
+import org.codehaus.groovy.runtime.IOGroovyMethods
+
+suite ("testAggQueryOnAggMV2") {
+    sql """ DROP TABLE IF EXISTS emps; """
```

Comment:
table name needs to be modified, the regression test is parallel, and the same table name will cause an error.

Reply:
> table name needs to be modified, the regression test is parallel, and the same table name will cause an error.

Those cases were placed in different directory and running on different databases, so it is ok.

Reply:
Sorry I didn't notice it, thanks for your remind. 😃